app/services/monitoring_service.py

```
@@ -0,0 +1,97 @@
+import logging
+from prometheus_client import REGISTRY
+import httpx
+from app.core.config import DISCORD_WEBHOOK_URL, APP_ENV
+
+logger = logging.getLogger(__name__)
+
+async def _acquire_distributed_lock(lock_key: str, ttl_seconds: int = 60) -> bool:
```

Ruff (0.15.2)

[warning] Unused function argument: `lock_key`

(ARG001)